torch/distributed/distributed_c10d.py

```diff
@@ -1712,8 +1708,12 @@ def broadcast_object_list(object_list, src=0, group=None):
             Each object must be picklable. Only objects on the ``src`` rank will
             be broadcast, but each rank must provide lists of equal sizes.
         src (int): Source rank from which to broadcast ``object_list``.
-        group: (ProcessGroup, optional): The process group to work on. If None,
-            the default process group will be used. Default is ``None``.
+        group (ProcessGroup, optional): The process group to work on. If
+            ``None``, the default process group will be used. Default is
+            ``None``.
+        device (torch.device): Device to load tensors contained in the received
+            object. If ``None``, tensors are loaded to their source device.
```

Comment:
nit: might be good to specify what source device here is. i.e. if rank 0 broadcasted tensors on GPU 0, then the tensors on rank 1 will be loaded on 0, right?